tests/python/gaia-ui-tests/gaiatest/mixins/gaia.py

```diff
@@ -21,6 +21,21 @@ class GaiaArguments(object):
          {'default': "undefined",
           'help': 'locale for the device, This value overrides the value from testvars.json file',
           }],
+        [['--capture'],
+         {'dest': 'capture',
+          'default': "whenfail",
```

Comment:
I'd turn it off be default. This can be a hassle if you're working on a failing test => your disk will be filled up without the user knowing the reason.